frontend/integration-tests/protractor.conf.ts

```diff
@@ -179,6 +183,18 @@ export const config = {
         password_manager_enabled: false,
       },
     },
+    'moz:firefoxOptions': {
+      binary: '/usr/bin/firefox',
```

Comment:
Is there a way we can make this work on other platforms like mac?

Reply:
We can do what Chrome side does;
```
binary: process.env.CHROME_BINARY_PATH,
```
Locally I see specifying a bogus path to `$FIREFOX_BINARY_PATH` would fail a run immediately, and not having the env var set would just go with "system default" which probably means the first `firefox` executable found on `$PATH`.

Reply:
@miyamotoh `$FIREFOX_BINARY_PATH` sounds good.

Reply:
This will be in my next force-push.